### DISCUSSION TITLE
Lazer strict tracking(mod) breaks the sliders

### DISCUSSION THREAD

**Discussion by seeeen11l1** [ppy/osu#23478] [Q&A] [answered]
After the update, strict tracking(ST) broke my sliders.  As in, the slider velocities are not what it should be. 
Logs:

[database.log](https://github.com/ppy/osu/files/11453681/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11453682/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11453683/network.log)
[performance.log](https://github.com/ppy/osu/files/11453684/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11453685/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11453686/updater.log)

**Comment by bdach** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/23471

**Reply by seeeen11l1** [reply]
Oh yeah sorry. I searched "slider tracking" in the board instead of "strict tracking" so I thought that it was undiscussed. My mistake. Won't happen again. Sorry.
